docs/app/data.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Why no TS here anymore?

Reply:
I felt the over-use of `any` defeated the purpose of TS :( 
faker didn't have types or this file was too dynamic -- I forgot the exact reason for the any